samples/cellular/modem_shell/src/sock/sock.c

```diff
@@ -653,6 +715,13 @@ int sock_open_and_connect(
 		}
 	}
 
+	if (secure && type == SOCK_DGRAM) {
+		err = sock_get_dtls_status(fd, session_cache, dtls_connection_id);
+		if (err) {
+			goto connect_error;
+		}
```

Comment:
If printing the DTLS status fails, the connection is still ok. Probably the error could be just ignored here. The function could even be made void.